src/library/dao/trackdao.cpp

```diff
@@ -392,8 +392,8 @@ void TrackDAO::addTracksPrepare() {
             "replaygain,"
             "replaygain_peak,"
             "wavesummaryhex,"
-            "timesplayed,"
-            "last_played_at,"
+            // timesplayed not written programatically (see comment)
```

Comment:
Rather than using commented-out lines, can you put a comment on the top saying something like "// This list should include all fields that are persisted to disk, and not those that are generated on-the-fly, like timesplayed and last_played_at.

Reply:
These 3 columns belong to the *play counter*. This is a very special case and I recommend to keep it close together.